Description: Don't install ca-bundle.crt from BOINC, and keep the system using
the ca-bundle.crt provided by ca-certificates
Author: Frank S. Thomas <frank@thomas-alfeld.de>
--- boinc.orig/client/Makefile.am
+++ boinc/client/Makefile.am
@@ -67,7 +67,7 @@
 boinc_client_LDADD = $(LIBBOINC) $(PTHREAD_LIBS)
 
 boinc_clientdir = $(bindir)
-boinc_client_DATA = ../curl/ca-bundle.crt
+#boinc_client_DATA = ../curl/ca-bundle.crt
 
 switcher_SOURCES = switcher.cpp
 
